spec/controllers/admin/api_controller_spec.rb

```diff
@@ -24,7 +24,7 @@
 
       it "redirects to the homepage" do
         get :index, params
-        it_redirects_to root_path
+        it_redirects_to_with_notice(root_path,"I'm sorry, only an admin can look at that area")
```

Comment:
In addition to Hound's comment about the comma, the descriptions for these two examples should ideally mention the flash notice.